modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java

```diff
@@ -81,22 +86,45 @@
 
     private List<NameValuePair> consentParameters = new ArrayList<>();
     private CookieStore cookieStore = new BasicCookieStore();
+    private final String username;
+    private final String userPassword;
+
+    @DataProvider(name = "configProvider")
+    public static Object[][] configProvider() {
+        return new Object[][]{{TestUserMode.SUPER_TENANT_ADMIN}, {TestUserMode.TENANT_ADMIN}};
+    }
+
+    @Factory(dataProvider = "configProvider")
+    public OAuth2ServiceAuthCodeGrantOpenIdTestCase(TestUserMode userMode) throws Exception {
+
+        context = new AutomationContext("IDENTITY", userMode);
+        this.username = context.getContextTenant().getTenantAdmin().getUserName();
+        this.userPassword = context.getContextTenant().getTenantAdmin().getPassword();
+    }
 
     @BeforeClass(alwaysRun = true)
     public void testInit() throws Exception {
-        super.init(TestUserMode.SUPER_TENANT_USER);
+
+        backendURL = context.getContextUrls().getBackEndUrl();
+        loginLogoutClient = new LoginLogoutClient(context);
+        sessionCookie = loginLogoutClient.login();
+        identityContextUrls = context.getContextUrls();
+        tenantInfo = context.getContextTenant();
+        userInfo = tenantInfo.getContextUser();
+
+        appMgtclient = new ApplicationManagementServiceClient(sessionCookie, backendURL, null);
+        adminClient = new OauthAdminClient(backendURL, sessionCookie);
+        remoteUSMServiceClient = new RemoteUserStoreManagerServiceClient(backendURL, sessionCookie);
 
         logManger = new AuthenticatorClient(backendURL);
-        adminUsername = userInfo.getUserName();
-        adminPassword = userInfo.getPassword();
-        String sessionIndex = logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-                isServer.getSuperTenant().getTenantAdmin().getPassword(),
-                isServer.getInstance().getHosts().get("default"));
+        String sessionIndex = logManger.login(username, userPassword,
```

Comment:
format this line

Reply:
This line is no longer there in this class.